docs/tools/prism-eslint-hook.js

```diff
@@ -0,0 +1,334 @@
+"use strict";
+
+const Prism = require("prismjs");
+const { Linter } = require("../../lib/api");
+const astUtils = require("../../lib/shared/ast-utils");
```

Comment:
How would this work for https://github.com/eslint/zh-hans.docs.eslint.org, where we don't have these modules?

Reply:
Oh... I don't think it will work as it is now.
@kecrily Do you have any ideas how to solve it?

Reply:
What do you think about changing this PR as below?

- `astUtils` is used only to get the regex, so it inlines the regex.
- Try `require("../../lib/api").Linter` and `require("eslint").Linter` and use the one that does not cause an error as the `Linter`.
- Change the install script of `zh-hans.docs.eslint.org` to install eslint as a dependency.

Reply:
Another option is to not do red underlines outside of the US site. We could check to see if these utilities exist, and if not, just don't do the linting.

Reply:
>Another option is to not do red underlines outside of the US site. We could check to see if these utilities exist, and if not, just don't do the lining.

I'd prefer this option.

Reply:
> Another option is to not do red underlines outside of the US site. We could check to see if these utilities exist, and if not, just don't do the linting.

I've made changes to this PR to make it so 👍